lessons/11-basic-data-structs/README.md

```diff
@@ -1 +1,10 @@
 # Basic Data Structures: Arrays and Hashes
+
+## Learning Goals
+
+## Resources
+[Arrays](notes/arrays.md)  
+[Account Generator Assignment](assignments/account-generator.md)  
+
+[Hashes](notes/hashes.md)  
+[Account Generator Continued](assignments/account-generator-cont.md)  
```

Comment:
I realize this is silly, but can we number the resources? I think it makes it easier when referencing it with someone -- plus you know, I like order :-)